core/modules/taxonomy/taxonomy.entity.inc

```diff
@@ -65,7 +65,7 @@ class TaxonomyTerm extends Entity {
    * This property is not loaded, but may be used to modify the term parents via
    * TaxonomyTerm::save().
    *
-   * The property can be set to an array of term IDs. An entry of 0 means this
+   * The property can be set to an array of term IDs. An entry of array(0) means this
```

Comment:
Now this line exceeds 80 characters. Sorry for being so pedantic, but if it's not me, someone else will point to that. :wink:

Reply:
Fixed!